newdle/templates/replied_email.html

```diff
@@ -0,0 +1,20 @@
+<body>
+  <p>Hi,</p>
+  <p>
+    <strong>{{ participant }}</strong> replied to "<strong>{{ title }}</strong>".
+  </p>
+  {%- if comment %}
+  <p>
+    Comment: {{ comment }}
+  </p>
```

Comment:
```suggestion
    <p>
      Comment: {{ comment }}
    </p>
```

indentation

Reply:
@ThiefMaster this converts into:

```
  <p>
    <strong>Guinea Pig</strong> replied to "<strong>adsfasfsdfadsfa</strong>".
  </p>
    <p>
      Comment: asdfsdfasdf
    </p>

  <p>Selected dates:</p>
```

breaks the indentation for some reason

Reply:
it does, but in an html email this doesn't matter!